bedrock/firefox/templates/firefox/os/_in_the_news.html

```diff
@@ -1,16 +0,0 @@
-{% set feed_links = firefox_os_feed_links(LANG) %}
```

Comment:
You can delete `firefox_os_feed_links` in `bedrock/firefox/templatetags/helpers.py` (and accompanying tests)